Code/Framework/AzToolsFramework/AzToolsFramework/UI/PropertyEditor/PropertyEditorAPI_Internals.h

```diff
@@ -326,6 +327,12 @@ namespace AzToolsFramework
                 }
             }
 
+            if (m_widget)
```

Comment:
If I understood right, the reason we don't call `GetWidget()` here is bc we don't need to do reset if the widget is not created?

Reply:
Yes, correct.